shared/src/business/entities/courtIssuedDocument/CourtIssuedDocumentTypeD.test.js

```diff
@@ -57,6 +57,26 @@ describe('CourtIssuedDocumentTypeD', () => {
       });
     });
 
+    it('should NOT have error message for past date if it is NOT a new document', () => {
+      const date = calculateISODate({
+        dateString: createISODateString(),
+        howMuch: -5,
+        unit: 'days',
+      });
+      const extDoc = CourtIssuedDocumentFactory({
```

Comment:
How do we know this document is not new?

Reply:
It already has a createdAt field.